File: pe/visualstudio/PE.FSharp/Problem0010.fs
namespace PE.FSharp.Problem10

open Microsoft.FSharp.Core 
open System
open Microsoft.FSharp.Core.Operators 
open FSharp.MyLibraries.Primes


module SumOfPrimes =

    let sumOfPrimesBelow n = 
        listOfPrimes n
        |> List.map int64 
        |> List.sum

    let answer = 
        sumOfPrimesBelow 2000000

File: pe/visualstudio/PE.CSharp/Problem0010.cs
using System.Linq;
using FSharp.MyLibraries;

namespace PE.CSharp
{
    public class Problem0010
    {
        public long SumOfPrimesBelow(int input)
        {
            return Primes
                .listOfPrimes(input - 1)
                .Select(x => (long)x)
                .Sum();
        }
    }
}
